### PR TITLE
#287 Event visitors pdf report.

### PR DESCRIPTION
**Pull request by floor114**
Resolves [github issue](https://github.com/pivorakmeetup/pivorak-web-app/issues/287)

### Description

Feature for creating event visitors report in admin part.

### Pre-merge checklist
 
- [x] The PR relates to a single subject with a clear title and description
- [x] Verify that feature branch is up-to-date with `development` (if not - rebase it). 

### Demonstration

jul 16 2017 6-28 pm

### CODE REVIEW

**Review by sveredyuk — COMMENTED** [review]
A little bit more specs for pdf? Check simplecov

**Comment by floor114** [comment]
@sveredyuk done :100: .

File: app/helpers/admin/events_helper.rb [hunk 1]
```diff
@@ -15,6 +15,14 @@ def admin_visit_requests_link(event, options = { class: 'item' })
         admin_event_visit_requests_path(event), options
     end
 
+    def admin_visitors_report_link(event)
+      return unless event.confirmation?
```
Comment (denysmedynskyi) [inline app/helpers/admin/events_helper.rb]: I am not sure about this condition. What if event already started(`live`) and we need to print names one more time?
However it is a minor issue, that can be checked with Oksana or Anna after the merge.

**Review by denys-medynskyi — APPROVED** [review]
Good job!
However, I am concerned with condition for showing
What if event already started(live) and we need to print names one more time?
Tell me what do you think about it and you can merge.

**Comment by floor114** [comment]
@denysmedynskyi Yes, I agree with you, we need to check It.

**Comment by sveredyuk** [comment]
@denysmedynskyi We always can use hand power )

**Pull request merged**